books/RayTracingTheRestOfYourLife.html

```diff
@@ -353,59 +466,93 @@
+we'll first need to know what a _density function_ is.
+
+A _density function_ is just the continuous version of a histogram. Here’s an example of a histogram
 from the histogram Wikipedia page:
 
   ![Figure [histogram]: Histogram example](../images/fig-3.03-histogram.jpg)
 
 </div>
 
 <div class='together'>
-If we added data for more trees, the histogram would get taller. If we divided the data into more
-bins, it would get shorter. A discrete density function differs from a histogram in that it
-normalizes the frequency y-axis to a fraction or percentage (just a fraction times 100). A
-continuous histogram, where we take the number of bins to infinity, can’t be a fraction because the
-height of all the bins would drop to zero. A density function is one where we take the bins and
-adjust them so they don’t get shorter as we add more bins. For the case of the tree histogram above
-we might try:
+If we added data for more trees, the number of bins would stay the same, but each would get taller.
+If we divided the data into more bins, we'd have more bins, but each would get shorter. If we took
+the number of bins and raised it infinity, we'd have an infinite number of zero-height bins. To
+solve for this, a discrete density function differs from a histogram in that it normalizes the
+y-axis to a fraction or percentage (just a fraction times 100) of the total. So a density function
+is a continuous histogram where all of the values are normalized against a total. For the case of
+the tree histogram above we could try to create a _density function_ by:
 
-  $$ \text{bin-height} = \frac{(\text{Fraction of trees between height }H\text{ and }H’)}{(H-H’)} $$
+    $$ \text{bin-height} = \frac{(\text{Fraction of trees between height }H\text{ and }H’)}
+                                {(H-H’)} $$
 </div>
 
 <div class='together'>
-That would work! We could interpret that as a statistical predictor of a tree’s height:
+That would work! If we had a specific tree we wanted to know the height of, we could create a
+_probability function_ that would tell us how likely it is for our tree to fall within a specific
+bin.
+
+    $$ \text{probability of Bin i} = \frac{\text{Number of trees in Bin i}}
+                                          {\text{Number of trees total}} $$
+
+If we combined our _probability function_ and our _density function_ we could interpret that as a
+statistical predictor of a tree’s height:
 
   $$ \text{Probability a random tree is between } H \text{ and } H’ = \text{bin-height}\cdot(H-H’)$$
 </div>
 
-If we wanted to know about the chances of being in a span of multiple bins, we would sum.
-
-A _probability density function_, henceforth _PDF_, is that fractional histogram made continuous.
+Indeed, with this continuous probability function we can now say the likelihood that a tree has a
+height that spans multiple bins. This is  _probability density function_, henceforth _PDF_, is a
```

Comment:
“This is a _probability density function_ (henceforth _PDF_): a continuous function that ...”